partner_priority/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Priority',
+    'summary': "Adds priority to partners.",
+    'author': "Patrick Wilson, Odoo Community Association (OCA)",
+    'website': "https://github.com/OCA/partner-contact",
+    'category': 'Extra Tools',
```

Comment:
The category Extra Tools does not really fit. Use "Customer Relationship Management" or "Partner Management".